src/Compilers/Core/Portable/EncodedStringText.cs

```diff
@@ -74,7 +86,7 @@ internal static SourceText Create(Stream stream, Encoding defaultEncoding = null
 
             try
             {
-                return Decode(stream, defaultEncoding ?? s_defaultEncoding, checksumAlgorithm, throwIfBinaryDetected: detectEncoding);
+                return Decode(stream, defaultEncoding ?? GetDefaultAnsiEncoding(), checksumAlgorithm, throwIfBinaryDetected: detectEncoding);
```

Comment:
Would this part impact our determinism deliverables?  It would seem to allow two compilations with the same command line to produce a different SHA depending on where they were run from.  

Although there are already other implicit imputs to the compiler (environment variables for example).  We have to rationalize those, perhaps we can just treat this the same way.